rapidtide/workflows/rapidtide_parser.py

```diff
@@ -210,686 +257,933 @@ def _get_parser():
+            "components."
+        ),
+        default=False,
+    )
+    experimental.add_argument(
+        "--dispersioncalc",
+        dest="dodispersioncalc",
+        action="store_true",
+        help=(
+            "Generate extra data during refinement to "
+            "allow calculation of dispersion."
+        ),
+        default=False,
+    )
+    experimental.add_argument(
+        "--acfix",
+        dest="fix_autocorrelation",
+        action="store_true",
+        help=(
+            "Perform a secondary correlation to "
+            "disambiguate peak location. Experimental."
+        ),
+        default=False,
+    )
+    experimental.add_argument(
+        "--alwaysmultiproc",
+        dest="alwaysmultiproc",
+        action="store_true",
+        help=("Use the multiprocessing code path even when nprocs=1."),
+        default=False,
+    )
+    experimental.add_argument(
+        "--tmask",
+        dest="tmaskname",
+        action="store",
+        type=lambda x: pf.is_valid_file(parser, x),
+        metavar="FILE",
+        help=(
+            "Only correlate during epochs specified "
+            "in MASKFILE (NB: each line of FILE "
+            "contains the time and duration of an "
+            "epoch to include."
+        ),
+        default=None,
+    )
 
     # Debugging options
-    debugging = parser.add_argument_group('Debugging options (not for general use)')
-    experimental.add_argument('--singleproc_getNullDist',
-                              dest='singleproc_getNullDist',
-                              action='store_true',
-                              help=('Force single proc path for getNullDist.'),
-                              default=False)
-    experimental.add_argument('--singleproc_calcsimilarity',
-                              dest='singleproc_calcsimilarity',
-                              action='store_true',
-                              help=('Force single proc path for calcsimilarity.'),
-                              default=False)
-    experimental.add_argument('--singleproc_peakeval',
-                              dest='singleproc_peakeval',
-                              action='store_true',
-                              help=('Force single proc path for peakeval.'),
-                              default=False)
-    experimental.add_argument('--singleproc_fitcorr',
-                              dest='singleproc_fitcorr',
-                              action='store_true',
-                              help=('Force single proc path for fitcorr.'),
-                              default=False)
-    experimental.add_argument('--singleproc_glm',
-                              dest='singleproc_glm',
-                              action='store_true',
-                              help=('Force single proc path for glm.'),
-                              default=False)
+    debugging = parser.add_argument_group("Debugging options")
+    debugging.add_argument(
```

Comment:
These arguments were under `experimental`, but based on their positions in the file I assumed they were actually debugging arguments.

Reply:
You are correct.  They should be in a debugging group.